security/security-core/src/main/java/io/camunda/security/configuration/TokenClaim.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.security.configuration;
+
+public class TokenClaim {
```

Comment:
:wrench: Even spring configuration properties can be records now :)

Reply:
Ah yes, good catch, I will update this